src/views/preview/preview.jsx

```diff
@@ -353,19 +304,40 @@ class Preview extends React.Component {
                         onUpdate={this.handleUpdate}
                     />
                 </Page> :
-                <IntlGUI
-                    enableCommunity
-                    hideIntro
-                    assetHost={this.props.assetHost}
-                    backpackOptions={this.props.backpackOptions}
-                    basePath="/"
-                    className="gui"
-                    projectHost={this.props.projectHost}
-                    projectId={this.state.projectId}
-                    projectTitle={this.props.projectInfo.title}
-                    onClickLogout={this.handleLogout}
-                    onUpdateProjectTitle={this.handleUpdateProjectTitle}
-                />
+                <React.Fragment>
+                    <IntlGUI
+                        enableCommunity
+                        hideIntro
+                        assetHost={this.props.assetHost}
+                        backpackOptions={this.props.backpackOptions}
+                        basePath="/"
+                        className="gui"
+                        projectHost={this.props.projectHost}
+                        projectId={this.state.projectId}
+                        projectTitle={this.props.projectInfo.title}
+                        // onClickLogout={this.handleLogout}
+                        onCloseAccountNav={this.props.handleCloseAccountNav}
+                        onCloseCanceledDeletion={this.props.handleCloseCanceledDeletion}
+                        onCloseLogin={this.props.handleCloseLogin}
+                        onCloseRegistration={this.props.handleCloseRegistration}
+                        onCompleteRegistration={this.props.handleCompleteRegistration}
+                        onLogIn={this.props.handleLogIn}
+                        onLogOut={this.props.handleLogOut}
+                        onOpenAccountNav={this.props.handleOpenAccountNav}
+                        onOpenRegistration={this.props.handleOpenRegistration}
+                        onToggleLoginOpen={this.props.handleToggleLoginOpen}
+                        onUpdateProjectTitle={this.handleUpdateProjectTitle}
+                    />
+                    <Registration />
+                    <LoginDropdown
```

Comment:
I think this is pretty funny that it works, but also seems brittle since the GUI can shift under www's feet, and it will be hard to ensure the dropdown displays as we expect in all configurations of the GUI. I think it'd be safer to pass this component to the GUI so the GUI can place it where it needs it.

Reply:
I'll take a crack at this.